pkg/bgpv1/agent/controller.go

```diff
@@ -7,11 +7,13 @@ import (
 	"context"
 	"fmt"
 	"runtime/pprof"
+	"time"
```

Comment:
I think now we are using cilium/pkg/time instead of std time pkg.

Reply:
I removed the retry logic, so this is no longer relevant.